bedrock/products/templatetags/misc.py

```diff
@@ -76,16 +80,14 @@ def vpn_subscribe_link(ctx, entrypoint, link_text, plan=None, class_name=None, l
     if plan in ['12-month', '6-month', 'monthly']:
 
         # Set a default plan ID using page locale. This acts as a fallback should geo-location fail.
-        if lang in ['de', 'fr']:
-            plan_default = settings.VPN_VARIABLE_PRICING[lang][plan]
-        else:
-            plan_default = ''
+        pricing_params = settings.VPN_VARIABLE_PRICING.get(lang, settings.VPN_VARIABLE_PRICING['us'])
+        plan_default = pricing_params[plan]['id']
+        plan_attributes = {}
 
         # HTML data-attributes are used by client side JS to set the correct plan ID based on geo.
-        plan_attributes = {
-            'data-plan-de': settings.VPN_VARIABLE_PRICING['de'][plan],
-            'data-plan-fr': settings.VPN_VARIABLE_PRICING['fr'][plan]
-        }
+        for country in settings.VPN_VARIABLE_PRICING.items():
```

Comment:
I usually set two variables when using `.items()` just for clarity and readability. e.g. `for country, attrs in settings.VPN_VARIABLE_PRICING.items():`. Not required, but a bit nicer than using just the tuple like this IMO.

Reply:
Nice suggestion! I'll update